### ISSUE TITLE
[CUDA Environment Update] Incompatible speficiations between CudaToolkit 10.0 and PyTorch 1.3.1

### ISSUE THREAD

**Issue by starg09** [facebookresearch/demucs#10]
Hello! While trying to setup I found the following error during the environment update step. Updating `environment-cuda.yml` so it required cudatoolkit 10.1 instead of 10.0 fixed it on my end, but thought about making a issue as to let you know!

```
(base) ###\demucs-master\demucs-master>conda env update -f environment-cuda.yml
Collecting package metadata (repodata.json): done
Solving environment: |
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed

UnsatisfiableError: The following specifications were found to be incompatible with each other:



Package cudatoolkit conflicts for:
pytorch=1.3.1 -> cudatoolkit[version='>=10.1,<10.2']
cudatoolkit=10.0

**Comment by adefossez** [comment]
Interesting. You are running on Windows? I put cudatoolikit 10.0 because 10.1 requires quite recent versions of the drivers, although it will definitely work with 10.1 too. Let me fix that.

**Comment by starg09** [comment]
Yeah, I'm running on Windows 10 (1909 Update).

**Comment by projectdrone2** [comment]
@starg09 , I had the same issue. To fix it, change 'cudatoolkit=10.0' to 'cudatoolkit=10.1.243' in environment-cuda.yml and run again 'conda env update -f environment-cuda.yml'. hope it helps

**Comment by adefossez** [comment]
It should work out of the box now, as I have updated the `environment-cuda.yml` :)
